src/scripts/components/Play.js

```diff
@@ -789,138 +796,144 @@ class Play extends Component<Props, State> {
+                      <TipOverlayWrapper>
+                        <TipOverlayContainer />
+                      </TipOverlayWrapper>
+                    ) : null}
+                  </PlayerWrapper>
+                </VideoCover>
+              </VideoWrapper>
+              {!isEmbed &&
+                video && (
+                <PlayInfo>
+                  {videoName && <Title small>{videoName}</Title>}
+                  {video.author && <Text>By {video.author}</Text>}
+                  {video.share && (
+                    <PlayInfoButtons>
+                      <ButtonIcon>
+                        <SVGIcon
+                          color="white"
+                          width="20px"
+                          height="20px"
+                          margin="0 20px 0 0"
+                          icon="#icon-play-view"
+                        />
+                          )
+                        <Text small gray>
+                          <TranslatedText message="player.views.zero" />
+                        </Text>
+                      </ButtonIcon>
+                      <ButtonIcon>
+                        <SVGIcon
+                          color="white"
+                          width="20px"
+                          height="20px"
+                          margin="0 20px 0 0"
+                          icon="#icon-play-like"
+                        />
+                        <Text small gray>
+                          <TranslatedText message="player.views.zero" />
+                        </Text>
+                      </ButtonIcon>
+                      <ButtonIcon>
+                        <SVGIcon
+                          color="white"
+                          width="20px"
+                          height="20px"
+                          margin="0 20px 0 0"
+                          icon="#icon-play-dislike"
+                        />
+                        <Text small gray>
+                          <TranslatedText message="player.views.zero" />
+                        </Text>
+                      </ButtonIcon>
+                    </PlayInfoButtons>
+                  )}
+                  <Text gray>
+                    <TranslatedText message="player.price" />{' '}
+                    <PlayInfoHighlight purple>
+                      {video.free ? (
+                        <TranslatedText message="player.free" />
+                      ) : (
+                        <TranslatedText message="player.free" />
+                      )}
+                    </PlayInfoHighlight>
+                  </Text>
+                  {video.description && (
+                    <DescriptionWrapper>
+                      <Text>{video.description}</Text>
+                    </DescriptionWrapper>
+                  )}
+                </PlayInfo>
+              )}
+            </VideoWrapperContent>
+            {!isEmbed && video && <SidebarTCR videoId={video && video.id} />}
```

Comment:
Can this logic be moved to a `this.renderTcrSidebar()` method? 

Also, which is the point of the `video && video.id` check if we already check `isEmbed && video`? Is that just to satisfy `flow`?